dgraphtest/local_cluster.go

```diff
@@ -551,7 +551,7 @@ func (c *LocalCluster) Upgrade(version string, strategy UpgradeStrategy) error {
 				return errors.Wrapf(err, "error during login before upgrade")
 			}
 		}
-		if err := hc.Export(DefaultExportDir); err != nil {
+		if err := hc.Export(DefaultExportDir, -1); err != nil {
```

Comment:
Can you add a comment on -1 as the namespace?

Reply:
I will do it as a separate PR. I have a few other changes that I need to merge too.